chef/cookbooks/ceph/recipes/osd.rb

```diff
@@ -71,7 +71,7 @@
   if is_crowbar?
     dirty = false
 
-    node.set["ceph"]["osd_devices"] ||= []
+    node.set["ceph"]["osd_devices"] = [] if node["ceph"]["osd_devices"].nil?
```

Comment:
Same question as is in https://github.com/crowbar/crowbar-ceph/pull/103. Why the change?